nordicsemi/zigbee/prod_config.py

```diff
@@ -0,0 +1,146 @@
+#
```

Comment:
Consider adding some unit tests for the implementation, as done in some other parts of the code.